### DISCUSSION TITLE
Multiple inverters, energy figures are only showing 1 inverters stats

### DISCUSSION THREAD

**Discussion by jdmevo123** [binsentsu/home-assistant-solaredge-modbus#208] [General]
I have two inverters, and the stats shown are only showing a single inverter, i.e. sensor.solaredge_dc_power only shows inverter one stats and not the including inverter 2.

How do I have all the stats aggregated into these values.

**Comment by mpredfearn** [comment]
To support multiple inverters, you should use https://github.com/WillCodeForCats/solaredge-modbus-multi/
